lte/gateway/c/session_manager/magma_logging.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#define MFATAL 0
+#define MERROR 1
+#define MWARNING 2
+#define MINFO 3
+#define MDEBUG 4
+
+// GLOG LOGGING
+#ifdef LOG_WITH_GLOG
+#include <glog/logging.h>
+
+#define MLOG(VERBOSITY) VLOG(VERBOSITY)
+#define MLOG_IF(VERBOSITY, CONDITION) VLOG_IF(VERBOSITY, CONDITION)
+
+#endif
+
+// NON GLOG LOGGING
+#ifndef LOG_WITH_GLOG
+#include <iostream>
+
+// for non glog use cases, just log to std cout
+struct _MLOG_NEWLINE {
+  ~_MLOG_NEWLINE() { std::cout << std::endl; }
+};
+#define MLOG(VERBOSITY)                                                        \
+  (_MLOG_NEWLINE(), std::cout << "[" << __FILE__ << ":" << __LINE__ << "] ")
+
+#endif
```

Comment:
isnt this a common library?

or are you placing this just for test?

Reply:
This I just placed until I figure out how to import it from common

Taking some shortcuts to get some sort of build first :)

Reply:
Well importing isn't the hard part. Making it work for bazel while not breaking the cmake build is harder😩

Reply:
well copying temporary there to allow a migration is fine. We can remove it later

Problem is if this happens with many more.

Is it possible to have hybrid builds, meaning, sessiond all using Bazel, but the rest still using cmake?

Reply:
should be yes! they are separate builds already, so that should not be a problem